ledger/accountdb.go

```diff
@@ -63,6 +63,12 @@ var accountsSchema = []string{
 	`CREATE TABLE IF NOT EXISTS accountbase (
 		address blob primary key,
 		data blob)`,
+	`CREATE TABLE IF NOT EXISTS onlineaccountbase (
```

Comment:
I think you may need to use the database migration system that was recently added: https://github.com/algorand/go-algorand/blob/master/ledger/acctupdates.go#L789

Reply:
given that this is a caching table ( i.e. it doesn't introduces a new data to the database but rather a computed value ), it doesn't need to affect the catchpoint files in any way. however, when restoring a catchpoint file, we need to handle this. You could follow the existing patten used for tables such as `assetcreators`.